### PR TITLE
Show full-size logo on slide first page

### PR DESCRIPTION
**Pull request by nsoranzo**
Broken in commit 379a80ab7dc2d589f58fda5221042947cb0c86aa .

### CODE REVIEW

File: _includes/default-header.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <nav class="navbar navbar-toggleable-md navbar-inverse">
         <div class="container">
             <a class="navbar-brand" href="{{ site.url }}/">
-                <img src="{{ site.url }}{{ site.logo }}" height="30" alt="Galaxy Training Network logo">
+                <img src="{{ site.url }}{{ site.small_logo }}" height="30" alt="Galaxy Training Network logo">
```
Comment (martenson) [inline _includes/default-header.html]: PR title says 'full size' but this says `small_logo`. Is this intended?

File: _layouts/base_slides.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
 <div class="my-footer"><span>
 {% if page.logo == "GTN" %}
-<img src="{{ site.url }}{{ site.logo }}" alt="Galaxy Training Network" style="height: 40px;"/>
+<img src="{{ site.url }}{{ site.small_logo }}" alt="Galaxy Training Network" style="height: 40px;"/>
```
Comment (martenson) [inline _layouts/base_slides.html]: PR title says 'full size' but this says small_logo. Is this intended?

Comment (nsoranzo) [inline _layouts/base_slides.html]: Yes, you need to look at line 41 of this file.

**Comment by willdurand** [comment]
Looks good to me!

File: _layouts/base_slides.html (hunk 2 above)
Comment (martenson) [inline _layouts/base_slides.html]: I see. Thanks!

**Pull request merged**